dotnet/TwitterBot/TwitterBotTests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("TwitterBotTests")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("TwitterBotTests")]
+[assembly: AssemblyCopyright("Copyright ©  2017")]
```

Comment:
I am not sure what is procedure for maintaining Copyright years...should it stay or bump if thing got improved today?

Reply:
Good catch, this is generated code so now we know when TwitterBot was first born :) I fixed two of these before merging but must have missed a project.